NaviGator/hardware_drivers/navigator_kill_board/nodes/kill_board_driver.py

```diff
@@ -89,12 +92,19 @@ def __init__(self):
 
         self._hw_kill_listener = AlarmListener("hw-kill", self.hw_kill_alarm_cb)
         self._kill_listener = AlarmListener("kill", self.kill_alarm_cb)
+        self.kill_broadcaster = AlarmBroadcaster("kill")
+        self.kill_broadcaster.wait_for_server()
+        self._network_kill_listener = AlarmListener(
+            "network-loss", self.network_kill_alarm_cb
+        )
         self._hw_kill_listener.wait_for_server()
         self._kill_listener.wait_for_server()
+        self._network_kill_listener.wait_for_server()
         rospy.Subscriber("/wrench/selected", String, self.wrench_cb)
-        rospy.Subscriber(
-            "/network", Header, self.network_cb
-        )  # Passes along network hearbeat to kill board
+        # rospy.Subscriber(
+        #     "/network", Header, self.network_cb
+        # )  # Passes along network hearbeat to kill board
```

Comment:
This should be removed or a comment should be added explaining why this is commented out.